dev/com.ibm.ws.jbatch.cdi.2.1/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
+   version="1.1" 
+   bean-discovery-mode="all">
```

Comment:
```suggestion
<beans xmlns="https://jakarta.ee/xml/ns/jakartaee"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd"
   version="4.0"
   bean-discovery-mode="all">
```

Reply:
This beans.xml does confuse me since all classes in the bundle are excluded by the exclude rule.  Is there a point to having the file?

Reply:
Not sure, I'll have to ask Scott or Dan tomorrow.